src/main/java/uk/gov/companieshouse/items/orders/api/model/DeliveryTimescale.java

```diff
@@ -20,6 +21,11 @@ public int getIndividualCertificateCost() {
         public int getExtraCertificateDiscount() {
             return SAME_DAY_EXTRA_CERTIFICATE_DISCOUNT;
         }
+
+        @Override
+        public ProductType getFirstCertificateProductType() {
+            return CERTIFICATE_SAME_DAY;
+        }
     };
 
     private static final int STANDARD_INDIVIDUAL_CERTIFICATE_COST = 15;
```

Comment:
Is it worth putting these in a properties file?

Reply:
Good question. My impression to date has been that these values don't change very often. However, I think this is really a question for Barri and Emma. I'll put it to them.

Reply:
Now discussed this point with Barri, Emma and Billy. They all feel it would be better to make these values somewhat configurable, and Billy has suggested that separate YAML file be used for this purpose. I will look into that.